src/main/java/uk/gov/hmcts/reform/sscs/robotics/RoboticsJsonMapper.java

```diff
@@ -253,6 +253,8 @@ private static String getCaseCode(SscsCaseData sscsCaseData) {
 
         if (ESA == benefit) {
             return ESA_CASE_CODE;
+        } else if (DLA ==  benefit) {
```

Comment:
Don't need this (or the carers allowance part which can be removed - sure i commented on a PR last week about that). This block is only put in for legacy cases so don't need to worry about it for new benefit types.